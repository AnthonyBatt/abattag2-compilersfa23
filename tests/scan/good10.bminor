'a''\a''\0x41'
