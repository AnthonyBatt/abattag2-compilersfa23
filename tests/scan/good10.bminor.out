TOKEN_CHAR_LITERAL        a
TOKEN_CHAR_LITERAL        
TOKEN_CHAR_LITERAL        A
